src/tooltip/tooltip.js

```diff
@@ -284,15 +280,16 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
             /**
              * Observe the relevant attributes.
              */
-            if (!options.useContentExp) {
-              attrs.$observe( type, function ( val ) {
+            attrs.$observe( type, function ( val ) {
+              $timeout(positionTooltip, 0, false);
```

Comment:
From what I understand, $timeout(fn, 0, false) is sometimes preferred over $applyAsync as it won't trigger a digest cycle.  There is also $evalAsync with a good explanation of the difference between $applyAsync [here](http://www.bennadel.com/blog/2751-scope-applyasync-vs-scope-evalasync-in-angularjs-1-3.htm)